ipatests/test_xmlrpc/test_host_plugin.py

```diff
@@ -100,6 +100,9 @@
 host_cert = get_testcert(DN(('CN', api.env.host), subject_base()),
                          'host/%s@%s' % (api.env.host, api.env.realm))
 
+missingrevzone = u'30.16.172.in-addr.arpa.'
```

Comment:
I don't understand the purpose of this and the subsequent attempt to delete it. Isn't the point that the reverse zone is missing? Or is it that the zone in fact does exist in the beginning of the test so you delete it? Is this because it would not be predictable to use some hardcoded value?

Will this have a potential impact on subsequent tests?

Reply:
The test needs to use a reverse zone that is not managed by IPA, and xmlrpc tests can be launched on an existing instance (on which I do not control the defined reverse zones), hence the deletion.

But I noted that this test fails on Travis because there is no DNS server answering for this reverse zone using a private IP address. I need to re-work this fix.